### PR TITLE
My request :3

### PR DESCRIPTION
**Pull request by l3gacyb3ta**
## Describe your project in 1-3 sentences

I am building a collapse resistant mesh network at my school.

## Hack Club Slack handle

@arky w rn! (Arcade Wise tho if i change it)

## Checklist before submitting proposal:

- [x] I am a current high school student (or younger).
- [X] I have submitted an RSVP at <https://hackclub.com/winter>
- [X] I am a member of the Hack Club Slack (Join at <https://hackclub.com/slack>)
- [X] I have self-reviewed my project proposal.

### CODE REVIEW

File: l3gacyb3ta.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+- [meshtastic](https://meshtastic.org/)
+- [disaster.radio](https://disaster.radio/)
+- [LOW←TECH MAGAZINE's solar website](https://solar.lowtechmagazine.com/about.html)
+
+## Plan
+
+1. Research actual networking solutions.
+1. Get a serial-based version working.
+1. Optimize power usage and online-offline transition
+1. Build a web interface.
```
Comment (exu3) [inline l3gacyb3ta.md:20]: What might you use to build this web interface? Could you add these details to your proposal?

File: l3gacyb3ta.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+1. Test at a larger scale.
+1. Deployment at my school!
+
+## Budget
+
+| Product                       | Supplier/Link                                                                                                                                                    | Cost    |
+|-------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------|
+| LILYGO® TTGO US(915MHz) x 3   | https://store.rokland.com/products/lilygo-ttgo-meshtastic-t-beam-v1-1-esp32-lora-915-mhz-wireless-module-wifi-gps-neo-6m-with-oled-display-for-arduino-q202-l206 | $128.30 |
```
Comment (exu3) [inline l3gacyb3ta.md:28]: Would you be needing to solder headers on this thing?

File: l3gacyb3ta.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+| TTL Serial USB Adapter        | https://www.amazon.com/DSD-TECH-Adapter-FT232RL-Compatible/dp/B07BBPX8B8/                                                                                        | $15.40  |
+| 5v 1.2w Solar Panel x 3       | https://www.adafruit.com/product/5368                                                                                                                            | $55.33  |
+| 3.3-6v to 3.3v Buck Converter | https://www.aliexpress.us/item/2251832868235446.html?gatewayAdapt=glo2usa4itemAdapt&_randl_shipto=US                                                             | $8.10   |
+| 18650 Batteries x 3           | https://www.amazon.com/Tokeyla-1%EF%BF%B58%EF%BF%B56%EF%BF%B55%EF%BF%B50-Rechargeabe-Batteries-Flashlights/dp/B0BG8CX93Q                                         | $24.99  |
+| Total                         |                                                                                                                                                                  | $232.12 |
```
Comment (exu3) [inline l3gacyb3ta.md:33]: Would you be able to explain what each of these components are for? It'd be great if you could add such an explanation to your `Plan`, describing how all the hardware is assembled.

File: l3gacyb3ta.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
+---
+name: "@l3gacyb3ta"
+project: "Collapse-resistant Mesh Networking"
+---
+
+# Project Name
```
Comment (exu3) [inline l3gacyb3ta.md:6]:
```suggestion
# Collapse-resistant Mesh Networking
```

**Review by exu3 — CHANGES_REQUESTED** [review]
Neat idea! I left some comments below. Let me know when you're ready for another review.

File: l3gacyb3ta.md (hunk 2 above)
Comment (l3gacyb3ta) [reply l3gacyb3ta.md:28]: Yes! I have the tools for that though

File: l3gacyb3ta.md (hunk 3 above)
Comment (l3gacyb3ta) [reply l3gacyb3ta.md:33]: Oh! Yeah sure

File: l3gacyb3ta.md (hunk 1 above)
Comment (l3gacyb3ta) [reply l3gacyb3ta.md:20]: Oh! Ok.

File: l3gacyb3ta.md (hunk 4 above)
Comment (l3gacyb3ta) [reply l3gacyb3ta.md:6]: fixed!

**Review by exu3 — APPROVED** [review]
Nice work! The @hackclub/bank-ops team will follow up with next steps here on receiving your grant.

Excited to see what you ship!

**Review by MelSmith104 — COMMENTED** [review]
hey @l3gacyb3ta this project sounds very doomsday and very interesting - i really can't wait to hear more! dm'ing you on slack right now to chat about next steps :)

**Review by MelSmith104 — APPROVED** [review]

**Pull request merged**